src/clients/api/mutations/useSupply.ts

```diff
@@ -1,21 +1,26 @@
 import { useMutation, MutationObserverOptions } from 'react-query';
-import { Bep20 } from 'types/contracts';
-import { TokenId } from 'types';
-import { queryClient, supply, ISupplyInput, SupplyOutput } from 'clients/api';
+import { VBep20 } from 'types/contracts';
+import { VTokenId } from 'types';
+import queryClient from 'clients/api/queryClient';
+import supply, { ISupplyInput, SupplyOutput } from 'clients/api/mutations/supply';
+
 import FunctionKey from 'constants/functionKey';
-import { useTokenContract } from 'clients/contracts/hooks';
+import { useVTokenContract } from 'clients/contracts/hooks';
+
+export type SupplyParams = Omit<ISupplyInput, 'tokenContract' | 'account'>;
 
 const useSupply = (
```

Comment:
I think we should use a more explicit name, like `useSupplyNonBnb`. Otherwise we have two reusable `useSupply` hooks in the codebase, it might get confusing to know which one to use